Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
IMO, using a dedicated library for URL parsing/manipulation is a good thing. (I did not suddenly just want to change this for fun, but simply because `http_build_query` caused trouble with the array syntax in query string.)

This does not increase coupling as the only parts affected are private functions.